README.md

```diff
@@ -8,6 +8,12 @@
 
 This Spring Boot Starter has been provided to help with integrating the [Nexmo Java SDK](https://github.com/Nexmo/nexmo-java) into your [Spring Boot](https://spring.io/projects/spring-boot) project.
 
+ * [Installation](#installation)
+ * [Usage](#usage)
+ * [Customizing the `NexmoClient`](#customization)
+ * [Customize Nexmo Client Version](#customization_version)
+ 
+<a name="installation"></a>
```

Comment:
HTML anchor tag is not needed. See [here](https://raw.githubusercontent.com/Vonage/vonage-java-sdk/master/README.md) for example on how to create TOC.